ext/scheduler/airflow/resources/base_dag.py

```diff
@@ -194,6 +195,21 @@
 {{- end -}}
 {{- end}}
 
+{{- range $_, $httpDependency := $.Job.ExternalDependencies.HTTPDependencies}}
+headers_dict_{{$httpDependency.Name}} = { {{- range $k, $v := $httpDependency.Headers}} '{{$k}}': '{{$v}}', {{- end}} }
```

Comment:
can't we just define a function & inline them the sensor itself, it will avoid these variables with suffixes of dependency names thoughts?

Reply:
as discussed, we will keep like this as of now